web/src/app/util/QuerySetFavoriteButton.js

```diff
@@ -56,19 +64,41 @@ export function QuerySetFavoriteButton(props) {
     variables: { id },
   })
   const isFavorite = data && data.data && data.data.isFavorite
-  const [toggleFav] = useMutation(mutation, {
+  const [showMutationErrorDialog, setShowMutationErrorDialog] = useState(false)
+  const [toggleFav, toggleFavStatus] = useMutation(mutation, {
     variables: {
       input: { target: { id, type: typeName }, favorite: !isFavorite },
     },
+    onError: () => setShowMutationErrorDialog(true),
   })
 
   return (
-    <SetFavoriteButton
-      typeName={typeName}
-      isFavorite={isFavorite}
-      loading={!data && loading}
-      onClick={() => toggleFav()}
-    />
+    <Grid>
```

Comment:
We can replace `<Grid>` with `<React.Fragment>`. 

https://reactjs.org/docs/fragments.html